addons/glusterfs/dev-guide/glusterfs-zh_CN.md

```diff
@@ -0,0 +1,142 @@
+# GlusterFS 接入 ThinRuntime 的简单示例
+
+## 前期准备
+glusterfs 接入 ThinRuntime 需要构造 GlusterFS-FUSE 客户端，本实例使用 [该项目](https://github.com/gluster/gluster-containers) 为基础构建 FUSE 镜像。
+
+## 准备 GlusterFS-FUSE 客户端镜像
+1. 挂载远程文件系统脚本
+
+在 FUSE 容器内需要提取相关的 **ThinRuntimeProfile、Dataset、ThinRuntime**资源中对远程文件系统的配置信息，相关信息以 JSON 字符串的方式保存到 FUSE 容器的 **/etc/fluid/config.json** 文件内。
+
+```python
+import json
+import os
+import subprocess
+
+obj = json.load(open("/etc/fluid/config.json"))
+
+mount_point = obj["mounts"][0]["mountPoint"]
+target_path = obj["targetPath"]
+
+os.makedirs(target_path, exist_ok=True)
+
+if len(mount_point.split(":")) != 2:
+    print(
+        f"The mountPoint format [{mount_point}] is wrong, should be server:volumeId")
+    exit(1)
+
+server, volume_id = mount_point.split(":")
+args = ["glusterfs", "--volfile-server", server, "--volfile-id",
+        volume_id, target_path, "--no-daemon", "--log-file", "/dev/stdout"]
+
+# Available options are described in the following pages:
+# https://manpages.ubuntu.com/manpages/trusty/en/man8/mount.glusterfs.8.html
+# https://manpages.ubuntu.com/manpages/trusty/en/man8/glusterfs.8.html
+if "options" in obj["mounts"][0]:
+    options = obj["mounts"][0]["options"]
+    for option in options:
+        if option[0] == "ro":
+            option[0] = "read-only"
+        elif option[0] == "transport":
+            option[0] = "volfile-server-transport" 
+            
+        if option[1].lower() == "true":
+            args.append(f'--{option[0]}')
+        elif option[1].lower() == "false":
+            continue
+        else:
+            args.append(f"--{option[0]}={option[1]}")
+
+subprocess.run(args)
+```
+该 shell 脚本创建挂载的文件夹并将远程文件系统挂载到目标位置（targetPath）。
+
+2. 创建 FUSE 客户端镜像
+
+
+将脚本打包入镜像。
+
+```dockerfile
+FROM gluster/glusterfs-client@sha256:66b1d51d327ab1c4c1d81e6bad28444e13e1746c2d6f009f9874dad2fba9836e
+
+ADD entrypoint.py /
+
+CMD ["python3", "/entrypoint.py"]
+```
+**由于 mount 命令会立即返回，为了保持该进程的持续运行（防止FUSE pod 反复重启），需要 sleep infinity 来保持进程的存在**。
```

Comment:
I think we don't need `sleep infinity` any more. Please remove it. Thanks.

Reply:
fixed